mxnet-bot/LabelBotFullFunctionality/LabelBot.py

```diff
@@ -135,16 +135,14 @@ def add_labels(self, issue_num, labels):
         :return Response denoting success or failure for logging purposes
         """
         labels = self._format_labels(labels)
-        issue_labels_url = 'https://api.github.com/repos/{repo}/issues/{id}/labels' \
-            .format(repo=self.repo, id=issue_num)
-
+        issue_labels_url = f'https://api.github.com/repos/{self.repo}/issues/{issue_num}/labels'
         response = requests.post(issue_labels_url, json.dumps(labels), auth=self.auth)
         if response.status_code == 200:
-            logging.info('Successfully added labels to {}: {}.'.format(str(issue_num), str(labels)))
+            logging.info(f'Successfully added labels to {issue_num}: {labels}.')
```

Comment:
What is the compatibility of this new `f`? Which version of Python support that?

Reply:
F-strings were introduced in Python 3.6. It is pre-parsed so it is faster and IMO is easier to read as well!